### ISSUE TITLE
Update Publications Page

### ISSUE THREAD

**Issue by msheskin** [lookit/lookit-api#1337]
# TL;DR
I'm not sure if this is the best place to collect this information, but as a first guess I am putting here new content for the CHS publications page https://childrenhelpingscience.com/publications/.

# Narrative
Back in 2020 when there were very few published papers using online data with children, we wanted to list all of them on CHS. Perhaps we want to abandon the effort now that there are a huge number and our list will definitely be incomplete, but in the meantime I will put here a new one a researcher sent me to add:

Reinecke, M. G., & Solomon, L. H. (2023). Children deny that God could change morality. Cognitive Development, 68, 101393. doi: 10.1016/j.cogdev.2023.101393

Here's a link to the free full text version:
https://columbiasamclab.weebly.com/uploads/5/9/0/6/59061709/reinecke_solomon_2023_cognitive_development_unformatted.pdf

And a link to the publisher's version:
https://www.sciencedirect.com/science/article/abs/pii/S0885201423000989

**Comment by becky-gilbert** [comment]
I'll leave the decision about whether to maintain this page to you, @mekline and @ianchandlercampbell. But just wanted to say that I'm in favor of updating it and would be happy to help maintain it. Adding new papers is an easy task ✅

**Comment by ianchandlercampbell** [comment]
I'd also speak in favor of maintaining this list-- thank you for volunteering @becky-gilbert! For the future, I might explore thinking about ways to organize this beyond "giant list of publications" (not a priority for now, though).

**Comment by ianchandlercampbell** [comment]
@becky-gilbert Agreed-- let's chat about this in stand-up tomorrow and we can make a game plan!

**Comment by becky-gilbert** [comment]
Update: 

I've added both of these papers to our [Zotero CHS citations group](https://www.zotero.org/groups/5819486/chs_lookit_citations), and they will soon appear on the [documentation site's publication page](https://lookit.readthedocs.io/en/master/publications.html) (when [this PR](https://github.com/lookit/lookit-docs/pull/456) is merged). But they won't appear on the lookit-api CHS Publications page until we update it. So I'll leave this issue open until then.

> We should also figure out how we want researchers to report new publications (online form?) and make a Slack announcement requesting updates to this list before we release it.

And Melissa is already on top of this 😄

**Comment by Ester0Awad** [comment]
Hi @becky-gilbert, I'd like to work on updating the publications list for this issue!

**Comment by becky-gilbert** [comment]
@Ester0Awad YAY! Thank you!! 🤩 🥇 🎉 ❤️ 

The way I did this on the documentation site was to put all publications into a Zotero group and load the list from there, which means we can update the list shown on the webpage by just adding a new paper to the group (this still requires a manual step rather than just picking up new papers automatically, but it's better than hard-coding everything into the page itself!). Here's the [JavaScript](https://github.com/lookit/lookit-docs/blob/develop/docs/source/_static/js/get_publication_list.js) that loads on the documentation's publication page, and here's the [publication page](https://github.com/lookit/lookit-docs/blob/develop/docs/source/publications.rst?plain=1) that contains the "zotero-publication-list" div that is updated by the JS. Feel free to re-use any of that for the [CHS publication page](https://github.com/lookit/lookit-api/blob/develop/web/templates/web/publications.html), or improve on it, or let me know if you have suggestions for other approaches. Thanks again!

**Comment by becky-gilbert** [comment]
Copying the last comment from the duplicate issue #1183 so that it doesn't get lost:

> Update: This has been implemented on the researcher-facing RTD site (https://github.com/lookit/lookit-docs/pull/456) but not on the CHS version of the publications page.
> 
> Questions for @mekline:
> 
> Do you want this same Zotero system to be implemented on the CHS publications page? (And if so, when?)
> The [CHS Publications page](https://childrenhelpingscience.com/publications/) exists but isn't linked anywhere on the site, other than a link that's buried in one of the FAQ items. So it's impossible to find unless you know the link. If we're going to maintain this page, should we also put it in the nav bar or link to it somewhere else?
